shard.yml

```
@@ -4,6 +4,6 @@ version: 0.1.0
 authors:
   - Brian J. Cardiff <bcardiff@manas.tech>
 
-crystal: 0.34.0
+crystal: 1.0.0
```

```suggestion
crystal: '>= 0.34.0'
```

Otherwise this PR removes compatibility with `0.34.x`, `0.35.x`, and `0.36.x`